.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 gradlew
-gradlew.bat
```

Comment:
We don't add .class files to the repository usually = best add a .class to the .gitignore.